atlast_sc/calculator.py

```diff
@@ -105,7 +106,7 @@ def bandwidth(self):
         return self.calculation_inputs.user_input.bandwidth.value
 
```

Comment:
Maybe add something to clarify this change.
```suggestion
    # With the new calculation of SEFD over the whole frequency range, SEFD is now dependent on bandwidth and so parameters must be updated every time bandwidth is changed.
```